cryptoxide/README.md

```diff
@@ -0,0 +1,79 @@
+# Cryptoxide
+
+[![Build Status](https://travis-ci.org/input-output-hk/rust-cardano.svg?branch=master)](https://travis-ci.org/input-output-hk/rust-cardano)
+![MIT or APACHE-2 licensed](https://img.shields.io/badge/licensed-MIT%20or%20APACHE--2-blue.svg)
+
+A pure-rust implementation of various modern cryptographic algorithms, which no dependencies
+and no foreign code (specially C or assembly), to work with WASM and embedded devices.
```

Comment:
A pure Rust implementation of various modern cryptographic algorithms, which has no dependencies
and no foreign code (specially C or assembly code). This is compatible with WASM and embedded devices.